docs/sdk-guide.md

```diff
@@ -684,6 +713,17 @@ Find example code at [CarbonReaderExample](https://github.com/apache/carbondata/
  public CarbonReaderBuilder withHadoopConf(Configuration conf);
 ```
 
+```
+  /**
+   * configure hadoop configuration with key value
```

Comment:
Change to "Updates the hadoop configuration with the given key value"

Reply:
ok, done